lib/rucio/core/rse.py

```diff
@@ -1219,8 +1219,28 @@ def del_protocols(rse_id, scheme, hostname=None, port=None, session=None):
                     i += 1
 
 
+MUTABLE_RSE_PROPERTIES = {
```

Comment:
@dchristidis @cserf 
I am wondering a bit about the `rse_type`, `deterministic` and `staging_area` attribute. Especially the first two if this is something we should allow to be changed? Opinions?

Reply:
That's more of a permission issue in ATLAS though. I am more wondering if these should be mutable at all, or if we just say that in these cases people need to make a new RSE. Especially for `rse_type`. Myself I am also a bit conflicted on it. I can see reasons where this makes sense, but especially the `deterministic` setting is dangerous, since this could corrupt the catalog.

Reply:
Any updates on this?

Reply:
Also keep in mind, that the `import_rses` uses the `update_rse` function. This functionality is used in the `rucio-admin` to import rses and update existing ones...

Reply:
I thought we discussed this in our followup 😄 Keep it how it works right now.